lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -254,6 +254,7 @@ module InternalConstants
 
     PLUGIN_NAME = 'Fluentd Google Cloud Logging plugin'.freeze
 
+    # Follows semver format.
```

Comment:
Want to reference semver.org to help explain the abbreviation (or just expand to "semantic versioning")?